package.json

```diff
@@ -232,5 +236,13 @@
   "bugs": {
     "url": "https://github.com/Trustroots/trustroots/issues"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "jest": {
+    "moduleNameMapper": {
+      "@/(.*)$": "<rootDir>/$1"
```

Comment:
This must have been a challenge to figure out! :open_mouth:

Reply:
Haha :) I usually have some other project to refer to for checking those... I think it took a while to setup jest for karrot at various points. Jest promises "configuration free"... but it's not really true... although it does a lot without any config.